### DISCUSSION TITLE
Time based position

### DISCUSSION THREAD

**Discussion by foxyfox** [gcormier/megadesk#90] [Ideas]
Hey,

it would be cool if I could set up a timer based positioning. So the desk changes from standing to sitting postion each hour or so and vice versa.

Best,
Andreas

**Comment by gcormier** [comment]
The ATTINY lacks an RTC to keep track of time. It could be hacked in with some code changes I imagine, but an external RTC would be a better option. Or, controlled via serial with an external device, as some others have done.

**Comment by AnAnalogGuy** [comment]
> Hey,
> 
> it would be cool if I could set up a timer based positioning. So the desk changes from standing to sitting postion each hour or so and vice versa.
> 
> Best, Andreas

Andreas, could you please elaborate a bit more on this idea? How should it work in detail? What settings would be needed?

Example: 
Automatically move between stored position 1 and stored position 2 every x minutes?
Functionality:
- toggle automatic move on/off
- choose intervall from a predefined list, i.e. 15min, 30min, 60min

Once started, shall the up/down intervall continue endless/until toggeld off, or shall it end automatically after i intervalls?
What shall happen, when the table is moved manually between two intervalls?
...

Any other aspects, ideas?
